From 6215e898232ba74a3676f87226a64501874fcbd2 Mon Sep 17 00:00:00 2001
From: Mike Perry <mikeperry-git@fscked.org>
Date: Tue, 30 Aug 2011 13:34:14 -0700
Subject: [PATCH 5/5] Smash the state.

What happened to you, Nederlanden? You used to be cool.
---
 security/manager/ssl/src/nsNSSCallbacks.cpp |    7 -------
 1 files changed, 0 insertions(+), 7 deletions(-)

diff --git a/security/manager/ssl/src/nsNSSCallbacks.cpp b/security/manager/ssl/src/nsNSSCallbacks.cpp
index 5e3a888..43e1c19 100644
--- a/security/manager/ssl/src/nsNSSCallbacks.cpp
+++ b/security/manager/ssl/src/nsNSSCallbacks.cpp
@@ -1065,13 +1065,6 @@ PSM_SSL_BlacklistDigiNotar(CERTCertificate * serverCert,
         }
       }
     }
-
-    // By request of the Dutch government
-    if (!strcmp(node->cert->issuerName,
-                "CN=Staat der Nederlanden Root CA,O=Staat der Nederlanden,C=NL") &&
-        CERT_LIST_END(CERT_LIST_NEXT(node), serverCertChain)) {
-      return 0;
-    }
   }
 
   if (isDigiNotarIssuedCert)
-- 
1.7.3.4

